src/operator/contrib/bounding_box.cc

```diff
@@ -47,23 +47,25 @@ the in_grad will be assigned 0.
 In summary, gradients are sticked to its boxes, will either be moved or discarded
 according to its original index in input.
 
-Input requirements:
-1. Input tensor have at least 2 dimensions, (n, k), any higher dims will be regarded
-as batch, e.g. (a, b, c, d, n, k) == (a*b*c*d, n, k)
-2. n is the number of boxes in each batch
-3. k is the width of each box item.
+Input requirements::
+
+  1. Input tensor have at least 2 dimensions, (n, k), any higher dims will be regarded
+  as batch, e.g. (a, b, c, d, n, k) == (a*b*c*d, n, k)
+  2. n is the number of boxes in each batch
+  3. k is the width of each box item.
 
 By default, a box is [id, score, xmin, ymin, xmax, ymax, ...],
-additional elements are allowed.
-- `id_index`: optional, use -1 to ignore, useful if `force_suppress=False`, which means
-we will skip highly overlapped boxes if one is `apple` while the other is `car`.
-- `coord_start`: required, default=2, the starting index of the 4 coordinates.
-Two formats are supported:
-  `corner`: [xmin, ymin, xmax, ymax]
-  `center`: [x, y, width, height]
-- `score_index`: required, default=1, box score/confidence.
-When two boxes overlap IOU > `overlap_thresh`, the one with smaller score will be suppressed.
-- `in_format` and `out_format`: default='corner', specify in/out box formats.
+additional elements are allowed::
+
+  - `id_index`: optional, use -1 to ignore, useful if `force_suppress=False`, which means
```

Comment:
I think maybe this should not be indented.
If you look at the preview it throws the text in a code block and the backticks don't render like they should.
http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13261/1/api/python/ndarray/contrib.html#mxnet.ndarray.contrib.box_nms